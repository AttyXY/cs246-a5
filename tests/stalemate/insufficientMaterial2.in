setup 
+ K e1
+ N h1
+ k e8
done
game human human
