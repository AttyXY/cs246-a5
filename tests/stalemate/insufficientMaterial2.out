WELCOME TO CHESS! PLEASE ENTER ONE OF THE FOLLOWING COMMANDS TO BEGIN:
- setup
- game [player1] [player2]

STARTING SETUP

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K---
   abcdefgh

8  --------
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--N
   abcdefgh

8  ----k---
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--N
   abcdefgh

STARTING GAME!

8  ----k---
7  --------
6  --------
5  --------
4  --------
3  --------
2  --------
1  ----K--N
   abcdefgh

Stalemate!
Final Score:
White: 0.5
Black: 0.5
THANKS FOR PLAYING CHESS!
